task_sdk/src/airflow/sdk/__init__.py

```diff
@@ -67,6 +67,7 @@
     "AssetAny": ".definitions.asset",
     "AssetWatcher": ".definitions.asset",
     "BaseOperator": ".definitions.baseoperator",
+    "BaseOperatorLink": ".definitions.baseoperatorlink",
```

Comment:
I think we should also have a matching import in the TYPE_CHECKING block.

Reply:
Good catch. Made that change